css-flexbox/display-flex.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - display: flex</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck,
+      .card-wrapper {
+        display: flex;
+        padding: 10px;
+      }
```

Comment:
These should be 2 separate rulesets. Only `#card-deck` should get `display: flex`. Both of them get the `padding`, but only one should get `display: flex`